spec/resources/subscription_spec.rb

```diff
@@ -99,4 +100,55 @@
       end
     end
   end
+
+  describe "#reactivate_subscription", :vcr do
+    subject(:reactivate_subscription) { subscription.reactivate_subscription }
+
+    let(:base_url_matcher) { Regexp.new(".chargify.com/subscriptions") }
```

Comment:
aqui vc pode fazer assim
```suggestion
    let(:url_matcher) { Regexp.new(".chargify.com/subscriptions/#{subscription.id}/reactivate.json") }
```
ai ja vai pegar dinamico do `let(:subscription)` q ja existe em cada contexto